lib/core/lib/core/models/opportunity/funding.tsp

```diff
@@ -20,6 +20,9 @@ using CommonGrants.Fields;
 )
 @example(Examples.Funding.allFields, #{ title: "All fields defined" })
 model OppFunding {
+  /** Details about the funding available for this opportunity that don't fit other fields */
+  details?: string;
+
```

Comment:
I added this because it could be used to store the raw text of details about opportunity funding when we can't parse them into `maxAwardAmount`, `minAwardAmount`, etc.